src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css

```diff
@@ -54,7 +54,5 @@
 }
 
 .accessible-view.hide {
-	position: fixed;
-	top: -2000px;
-	left:-2000px;
+	opacity: 0;
```

Comment:
Does `visibility: hidden` work? You'll also want pointer-events: none in case someone tries to click that area.

Reply:
`visibility: hidden` does not work. It did not work before that change. I tested today in insider's and realized that.